lib/tasks/taxonomy.rake

```diff
@@ -30,4 +36,21 @@ namespace :taxonomy do
     Rake::Task["taxonomy:create_taxons"].invoke
     Rake::Task["taxonomy:link_taxons"].invoke
   end
+
+  def parse_base_paths_string(base_paths)
+    pair_of_paths = base_paths.split('|')
+
+    unless pair_of_paths.all? { |pair| (pair.split(',').size % 2).zero? }
+      raise ArgumentError, base_paths_error_message
+    end
+
+    pair_of_paths.map do |pair|
+      splited_pair = pair.split(',')
+      { from: splited_pair.first, to: splited_pair.last }
+    end
+  end
+
+  def base_paths_error_message
+    "base_paths is expected to be composed of two comma-separated values, like so: '/path1,/path1-rename|/path2,/path2-rename,...'"
```

Comment:
Maybe - `base_paths should be a set of pairs, delimited by a pipe character, like so: ...`